composer.json

```diff
@@ -110,19 +122,19 @@
         }
     },
     "scripts": {
+        "post-autoload-dump": [
+            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
+            "@php artisan package:discover --ansi"
+        ],
         "post-root-package-install": [
             "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
         ],
         "post-create-project-cmd": [
-            "@php artisan key:generate"
-        ],
-        "post-autoload-dump": [
-            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
-            "@php artisan package:discover"
+            "@php artisan key:generate --ansi"
         ],
         "post-install-cmd": [
             "captainhook install -f",
-            "@php artisan passport:keys || @php -r \"return 0;\""
+            "@php artisan passport:keys || php -r \"return 0;\""
```

Comment:
What's the reason to change the second `@php` for `php` ?

Reply:
I was getting a weird error when running the script in the docker exec, removing the @ fixed it for me.

Reply:
mm interesting 🤔 what docker exec lines were you running exactly? I would like to try to reproduce

Reply:
<img width="929" alt="image" src="https://user-images.githubusercontent.com/30173059/225618099-8fbc531d-bc7a-402a-8695-f1502a74da9c.png">

This is the error I get even without running it in docker.